pkg/process/checks/checks.go

```diff
@@ -24,12 +22,19 @@ const (
 	ProcessEventsCheckName = "process_events"
 )
 
+// SysProbeConfig provides access to system probe configuration
+type SysProbeConfig struct {
```

Comment:
Why do you define this separate type instead of just using the system probe directly?

Reply:
So the unfortunate part is that system probe config is a struct defined in `cmd/system-probe`, effectively we'd be importing something outside of `pkg` which is not ideal. Waiting on https://github.com/DataDog/datadog-agent/pull/14024, which introduces `github.com/DataDog/datadog-agent/comp/core/sysprobeconfig` and `sysprobeconfig.Component`.

Reply:
Makes sense!